modules/mergeday/files/requirements.txt

```diff
@@ -1,2 +1,5 @@
 # python_version: 27
+attrs==19.1.0
+enum34==1.1.6
+mozilla-version==0.3.4
```

Comment:
This is now needed because of https://bugzilla.mozilla.org/attachment.cgi?id=9076779